implementations/rust/ockam/ockam/src/error.rs

```diff
@@ -0,0 +1,17 @@
+use ockam_core::Error;
+
+#[derive(Clone, Copy, Debug)]
+pub enum OckamError {
+    None,
+}
+
+impl OckamError {
+    pub const DOMAIN_CODE: u32 = 10_000;
```

Comment:
Why do we need it?

Reply:
I agree. How is this better than what Sanjo had before?